openzwave-sys/src/notification.rs

```diff
@@ -49,6 +49,21 @@ pub enum NotificationCode {
     Code_Alive
 }
 
+use std::fmt;
+impl fmt::Display for NotificationCode {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.pad(match self {
+            &NotificationCode::Code_MsgComplete => "Code_MsgComplete",
+            &NotificationCode::Code_Timeout => "Code_Timeout",
+            &NotificationCode::Code_NoOperation => "Code_NoOperation",
+            &NotificationCode::Code_Awake => "Code_Awake",
+            &NotificationCode::Code_Sleep => "Code_Sleep",
+            &NotificationCode::Code_Dead => "Code_Dead",
+            &NotificationCode::Code_Alive => "Code_Alive",
+        })
+    }
+}
```

Comment:
I think you can do like I did in https://github.com/fxbox/openzwave-rust/blob/2e8b44683d65b7020dd5239a886965e764c4212a/src/value_classes/value_id.rs#L102-L106 by reusing the Debug default implementation.